src/library/masq.js

```diff
@@ -380,6 +388,10 @@ class Masq {
       }
 
       this.hub = signalhub(channel, HUB_URLS)
+      this.hub.on('error', () => {
+        dispatchMasqError(MasqError.REPLICATION_SIGNALLING_ERROR)
```

Comment:
I think we should use SIGNALLING_SERVER_ERROR as this is not the instance used for replication

Reply:
It should be used for replicating the profile DB, so I think it's the same error